app/styleguide/HelloWorld.guide.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { SGPage, SGSection } from 'reapp-dev-tools';
+
+
+import HelloWorld from 'components/HelloWorld';
+
+export default class HelloWorldGuideComponent extends React.Component {
+    render() {
+        return (
+            <SGPage>
+
+                <SGSection title="HelloWorld - without properties">
+                    <HelloWorld />
+                </SGSection>
+
+                <SGSection title="HelloWorld - with value property">
+                    <HelloWorld value="Content for the component" />
```

Comment:
1. there is no `value` property in the `HelloWorld` component
2. you should try to use the properties `text` and `name` that you have implemented